### PR TITLE
Feat/agg pools

### LINKED ISSUE #435: Add new pool list (aggregator view)
```
poolType
address
poolTypeVersion
tokens (orderBy: index) {
  address
  decimals
  weight
}
root3Alpha
alpha
beta
c
s
lambda
tauAlphaX
tauAlphaY
tauBetaX
tauBetaY
u
v
w
z
dSq
```

filters:
```
swapEnabled
totalLiquidity_gt - used to limit pools and also to find top pools per token
totalShares_not_in: ["0", "0.000000000001"]
```

### PR DESCRIPTION
**Pull request by franzns**

### CODE REVIEW

File: modules/pool/lib/pool-gql-loader.service.ts [hunk 1]
```diff
@@ -541,6 +552,69 @@ export class PoolGqlLoaderService {
...
+                    ...poolWithoutTypeData,
+                    ...mappedData,
+                    ...(typeData as FxData),
+                };
+        }
+
+        return {
+            ...poolWithoutTypeData,
+            ...mappedData,
+        };
```
Comment (gmbronco) [inline modules/pool/lib/pool-gql-loader.service.ts:615]:
```suggestion
        return {
            ...pool,
            dynamicData: this.getPoolDynamicData(pool),
            poolTokens: pool.tokens.map((token) => this.mapPoolToken(token, token.nestedPool !== null)),
        };
```

File: modules/pool/pool.gql [hunk 2]
```diff
@@ -285,6 +296,145 @@ enum GqlPoolType {
...
+    """
+    version: Int!
+    """
+    Returns all pool tokens, including BPTs and nested pools if there are any. Only one nested level deep.
+    """
+    poolTokens: [GqlPoolTokenDetail!]!
+    """
+    Dynamic data such as token balances, swap fees or volume
+    """
+    dynamicData: GqlPoolDynamicData!
```
Comment (gmbronco) [inline modules/pool/pool.gql:355]: I suggest we return only what's needed. Currently there is a lot of "UI" stats data that is not needed and is cluttering the type. Same for token lists and nested pools. Skipping some "included" table queries will make it faster.

File: modules/pool/pool.service.ts [hunk 3]
```diff
@@ -84,6 +85,10 @@ export class PoolService {
         return this.poolGqlLoaderService.getPools(args);
     }
 
+    public async getAggregatorPools(args: QueryPoolGetPoolsArgs): Promise<GqlPoolAggregator[]> {
+        return this.poolGqlLoaderService.getAggregatorPools(args);
+    }
+
```
Comment (gmbronco) [inline modules/pool/pool.service.ts:91]: i'm thinking if it's not a good opportunity to refactor this service as well, make it similar to events, where we don't have a big service, but purpose specific functions that can be run independently.

Comment (franzns) [reply modules/pool/pool.service.ts:91]: I thought we ended up agreeing that we'll reuse what we have and do a bigger refactor early next year when we investigate auto generated queries

File: modules/pool/pool.gql (hunk 2 above)
Comment (franzns) [reply modules/pool/pool.gql:355]: I was just trying to reuse as much as possible as we will refactor the whole thing early next year

Comment (gmbronco) [reply modules/pool/pool.gql:355]: i had an issue locally where the query stalled when not specifying any chain, let's add a default limit at least, so the query doesn't hang.

**Pull request merged**